### PR TITLE
Merged master and resolved conflicts

### PR DESCRIPTION
**Pull request by bvaughn**
Merged master. Stack seems to have been deleted. 😆 

Search-and-replaced a bunch of markdown links to remove `/react/` prefixes

### CODE REVIEW

File: docs/docs/reference-react-component.md [hunk 1]
```diff
@@ -114,16 +114,16 @@ You can also return multiple items from `render()` using an array:
...
+    <li key="B">Second item</li>,
+    <li key="C">Third item</li>,
   ];
 }
 ```
 
 > Note:
 >
-> Don't forget to [add keys](/docs/lists-and-keys.html#keys) to elements in a fragment to avoid the key warning.
+> Don't forget to [add keys](https://facebook.github.io/react/docs/lists-and-keys.html#keys) to elements in a fragment to avoid the key warning.
```
Comment (gaearon) [inline docs/docs/reference-react-component.md]: ?

Comment (bvaughn) [reply docs/docs/reference-react-component.md]: Huh. No idea.

Comment (bvaughn) [reply docs/docs/reference-react-component.md]: That came from master.

https://raw.githubusercontent.com/facebook/react/master/docs/docs/reference-react-component.md

Comment (bvaughn) [reply docs/docs/reference-react-component.md]: This came in from `master` - looks like it was accidentally added via PR #10824

I've fixed it in this branch.

**Pull request merged**